app/controllers/hq/users_controller.rb

```diff
@@ -6,4 +6,24 @@ def index
     @users = User.all
   end
 
+  def edit
+    @user = User.find(params[:id])
```

Comment:
I am also a fan of leaving the single line here, unless the complexity grows or the code is repeated, then definitely abstracting it